test/fixtures/dialects/ansi/insert_b.yml

```diff
@@ -1,19 +0,0 @@
-# Insert Statements
```

Comment:
why did this file get deleted?

Reply:
It’s not used. There is no equivalent sql file and tests pass without it. I figured it was an old file that was no longer needed. It also fails yamllint at the mo but didn’t investigate how to fix as not needed.